test/Passes/tile-and-fuse-default.mlir

```diff
@@ -0,0 +1,274 @@
+// RUN: tpp-opt %s -split-input-file -tile-consumer-and-fuse-producers -cse | FileCheck %s
```

Comment:
Add `tileSize={1,1}` here and create a new small test for the default of 32 elsewhere.